sonic-net/sonic-gnmi
# sonic-gnmi: Master Arbitration

Adding doc for Master Arbitration HLD

Signed-off-by: sallylyu@google.com



#### Why I did it
Demonstrate the high level design of Master Arbitration feature.

#### How I did it
Provide the markdown document for the high level design.

#### How to verify it
We designed a set of unit test and system test cases to verify the Master Arbitration is working as expected. 

#### Which release branch to backport (provide reason below if selected)




N/A. Master Arbitration high level design is not a fix. No backporting is needed.

#### Description for the changelog

No change in changelog

#### Link to config_db schema for YANG module changes

No change in config_db schema.

#### A picture of a cute animal (not mandatory but encouraged)
![piggy](https://user-images.githubusercontent.com/108024270/224922490-039a01ab-60da-4278-968d-53c82ed8afff.jpeg)
